plugins/ssl/letsencrypt_weekly

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+#
+# monitor the number of CSRs by week for /etc/letsencrypt/csr/
+#
+# andreas perhab - andreas.perhab@wt-io-it.at
+# https://www.wt-io-it.at/
+#
+# License: GPLv2
+#
+# Parameters:
+#
+#  config
+#  autoconfig
+#
+#%# family=auto
+#%# capabilities=autoconf
+
+if [ "$1" == "autoconf" ] ; then
+    find /etc/letsencrypt/csr/ > /dev/null 2>&1 && echo "yes" || echo "no"
+elif [ "$1" == "config" ] ; then
+	echo "graph_title Letsencrypt certificates last week"
+	echo "graph_args --base 1000"
+	echo "graph_vlabel Number of certificates"
+	echo "graph_category certificates"
+	echo "letsencrypt_weekly.label Letsencrypt certificates last week"
+	echo "letsencrypt_weekly.warning :40"
```

Comment:
How about `warning=${warning:-:40} print_warning "letsencrypt_weekly"`?

Reply:
added it here as well